test/unit/refresh.js

```diff
@@ -994,6 +993,26 @@ describe('Unit tests for swiftserver:refresh', function () {
           assert(error.match(/Getting server SDK.*failed/), 'Thrown error should be about failing to download server SDK, it was: ' + error)
         })
       })
+
+      describe('SDKGen package format is incorrect', function () {
+        var runContext
+        var error
+        before(function () {
+          mockSDKGen.mockServerSDKPackageRequest(serverSDKName)
+          runContext = helpers.run(refreshGeneratorPath)
+                              .withOptions({ specObj: spec })
+          return runContext.toPromise().catch((e) => { error = e })
+        })
+
+        after(function () {
+          nock.cleanAll()
+          runContext.cleanTestDirectory()
+        })
+
+        it(`should throw error`, function () {
+          assert(error, 'SDKGEN Package dependency format is incompatible')
```

Comment:
Add should throw error assert and check the error message in second assert with REGEX.